frontend/explorer/public/graphs.html

```diff
@@ -22,7 +22,7 @@
 	</div>
 
 	<div class='banner'>
-		<h1 class='banner-head'>ThreeFold Chain Block Explorer</h1>
```

Comment:
we never said to remove the class. The class should be kept for css purposes, you'll need to give it a unique ID name such as 'page-title'.